docs/src/index.md

```diff
@@ -1,14 +1,15 @@
 # PowerSystems.jl
 
 ```@meta
-CurrentModule = SIIP_PACKAGE
+CurrentModule = SiennaTemplate
 ```
 
 ## Overview
 
-`PSIIP-PACKAGE .jl` is a [`Julia`](http://www.julialang.org) package that provides blah blah
+`SiennaTemplate.jl` is a [`Julia`](http://www.julialang.org) package that provides blah blah
+
+* * *
 
-------------
 SIIP-PACKAGE  has been developed as part of the Scalable Integrated Infrastructure Planning
```

Comment:
Can we replace this with the same About stub from power systems?

![Screenshot 2024-11-08 at 2 03 05 PM](https://github.com/user-attachments/assets/13215dd9-8ec7-4015-a874-10d425c1b330)